esp/esp/users/migrations/0015_auto_20180814_1725.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
I believe you'll need to regenerate this after merging with #2591, so they don't cause any conflicts.